src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -200,19 +200,25 @@ internal sealed partial class ConsoleHost
                 // The JumpList does persist as long as the filepath of the executable does not change but there
                 // could be disruptions to it like e.g. the bi-annual Windows update, we decided to
                 // not over-optimize this and always create the JumpList as a non-blocking background task instead.
-                Task.Run(() =>
+                try
                 {
-                    try
+                    // APIs are STA only and until issue 7216 is not resolved, PowerShell operates still in MTA mode.
+                    var thread = new Thread(() =>
                     {
-                        TaskbarJumpList.CreateElevatedEntry(ConsoleHostStrings.RunAsAdministrator);
-                    }
-                    catch
-                    {
-                        // Sporadic failures have been observed in some environments. Since the JumpList persists once it
-                        // has been registered, there is no harm suppressing the occasional failure.
-                    }
+                        try
+                        {
+                            TaskbarJumpList.CreateElevatedEntry(ConsoleHostStrings.RunAsAdministrator);
+                        }
+                        catch { }
```

Comment:
Could you please add a logging here?

Reply:
@daxian-dbw Can you please advice which method on `PSEtwLog` I should call and ideally how. I could've used some method and added an `EventId` but I'm aware that I'd have likely chosen a method that you wouldn't prefer. Some help would be appreciated.

Reply:
Hmm, I'm not sure if we care about the exceptions thrown out here. If we are able to catch, they are not fatal. Writing an ETW event at startup time is expensive, even if it's happening on a different thread.
What we want to achieve here? Write out an event can only be useful if `CreateElevatedEntry` start to fail consistently. But that can be easily noticed because the `Run as Administrator` option won't be generated for a pwsh instance.